SingularityService/src/main/java/com/hubspot/singularity/smtp/MailTemplateHelpers.java

```diff
@@ -34,6 +37,8 @@
   private static final String REQUEST_LINK_FORMAT = "%s/request/%s";
   private static final String LOG_LINK_FORMAT = "%s/task/%s/tail/%s";
   private static final String DEFAULT_TIMESTAMP_FORMAT = "MMM dd h:mm:ss a zzz";
+  private static final Optional<String> DEFAULT_TASK_LOG_ERROR_REGEX = Optional.of("ERROR");
+  private static final Long MAX_LOG_SEARCH_OFFSET = 1000000L;
```

Comment:
I am very open to ideas on the best value for this constant (the maximum number of bytes to search through for an error before giving up and showing the bottom of the file) as well. 1,000,000 bytes was completely arbitrary and isn't necessarily the best value. We can put this into the smtp config too if we would like, but we'll still need to choose a default.